common/tools/dev-tool/src/commands/samples/checkNodeVersions.ts

```diff
@@ -224,7 +224,7 @@ export const commandInfo = makeCommandInfo(
     "node-versions": {
       kind: "string",
       description: "A comma separated list of node versions to use",
-      default: "8,10,12"
+      default: "12,14,15"
```

Comment:
Question about node 10 - if it EOL on 4/30/2021 don't we want to continue supporting it for 6 months (so until 10/30)?

Reply:
fair enough, I added it back.